.github/workflows/publish-packages.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish packages
+
+on:
+  push:
+    branches: [v2]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Check out, and set up the node/ruby infra
+      - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+        with:
+          # Fetch the full history, to build attribution.json
+          fetch-depth: 0
+      - uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+        with:
+          node-version: "18.x"
+          registry-url: "https://registry.npmjs.org/"
+          cache: yarn
+
+      - name: Prepare website v2
+        run: |
+          yarn install
+          yarn docs-sync pull microsoft/TypeScript-Website-localizations#main 1
+          yarn bootstrap
+          yarn workspace typescriptlang-org setup-playground-cache-bust
+          yarn build
+        env:
+          YARN_CHECKSUM_BEHAVIOR: ignore
+
+      - name: Build website v2
+        run: |
+          yarn build-site
+          cp -r packages/typescriptlang-org/public site
+
```

Comment:
I'm not sure how much of this is needed to get valid packages, but I'm not taking any chances until we can rework this.